proxy/modules/proxy/install.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -eo pipefail
+
+SOURCES_DIR=/tmp/artifacts
+LICENSE_DIR=/root/licenses
+PRODUCT_LICENSE_DIR=${LICENSE_DIR}/${COM_REDHAT_COMPONENT}
+
+# Add kroxylicious user with UID 1001
+# The user is in the group 0 to have access to the mounted volumes and storage
+useradd -r -m -u 1001 -g 0 kroxylicious
+
+# untar the artifact containing license information
+mkdir -p ${PRODUCT_LICENSE_DIR}
+mv ${SOURCES_DIR}/kroxylicious-app-licenses.tar.gz ${LICENSE_DIR}
+tar -xvzf ${LICENSE_DIR}/kroxylicious-app-licenses.tar.gz -C ${PRODUCT_LICENSE_DIR} --no-same-owner
+
+# create destination folder of scripts, jars and config
+mkdir -p ${KROXYLICIOUS_HOME}
+
+# unzip archive/artifact with bridge scripts and jars
```

Comment:
nit
```suggestion
# unzip archive/artifact with proxy scripts and jars
```

Reply:
Also why use zip here at all and not tgz?

Reply:
The remainder of AMQ Streams uses zips for the licence artefact.  Consistency won out.